### PR TITLE
Dashboard page

### PR DESCRIPTION
**Pull request by consoledot**

### CODE REVIEW

**Comment by consoledot** [comment]
@Aliemeka pls can you help check the log, its building locally

**Comment by Aliemeka** [comment]
Screenshot 2023-09-19 at 11 36 41

File: src/components/activeWebinarCard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import { ClashDisplay } from "@/utils/font";
```
Comment (Aliemeka) [inline src/components/activeWebinarCard/index.tsx:1]: Change the spelling of this folder to ActiveWebinarCard

Comment (Aliemeka) [reply src/components/activeWebinarCard/index.tsx:1]: That's what causing the error

**Review by Aliemeka — APPROVED** [review]

**Pull request merged**